docs/architecture/key-concepts.md

```diff
@@ -18,45 +26,27 @@ Blocks can be static or dynamic. Static blocks contain rendered content and an o
-
-## Reusable blocks
-
-Reusable blocks is a block (or multiple blocks) that you can insert, modify, repeatable piece of content.
-
-The content and style of a reusable block is intended to be consistent wherever it is used. 
-
-Examples of reusable blocks include a block consisting of a heading whose content and a custom color that would be appear on multiple pages of the site and sidebar widgets that would appear on every page (widgets are planned to be available, but not yet possible, in Gutenberg). 
+A reusable blocks is a block (or multiple blocks) that can be inserted and edited globally at once. If a reusable block is edited in one place, those changes are reflected across all posts and pages that that block is used. Examples of reusable blocks include a block consisting of a heading whose content and a custom color that would be appear on multiple pages of the site and sidebar widgets that would appear on every page. 
 
 Any edits to a reusable block will appear on every other use of that block, saving time from having to make the same edit on different posts. 
 
-Reusable blocks are stored as a hidden post type (wp_block) and are dynamic blocks that "ref" or reference the post_id and return the post_content for that block. 
-
-The same reusable block can be used across different post types (e.g. post and page). 
+In technical details, reusable blocks are stored as a hidden post type (`wp_block`) and are dynamic blocks that "ref" or reference the `post_id` and return the `post_content` for that block.
 
-If you need to create a structure (a block consisting of heading, paragraph, and list) that is very similar across multiple posts but the content is slightly different across those pages or posts, you can do the following to minimize the amount of duplicate work to do:
+## Patterns
 
-1. create a 'skeleton' that will have shared characteristics (e.g. the same color background, font size)
-1. save this as a reusable block.
-1. Then, on other pages/posts:
-1. Within the block editor: insert the reusable block 
-1. Open the block's properties (three dots)
-and "convert to regular block"; the block is no longer 'reusable' and all edits to this block will only appear on this page/post.
+A block pattern is a group of blocks that have been combined together creating a design pattern. These design patterns provide a starting point for building more advanced pages and layouts quickly. A block pattern can be as small as a single block or as large as a full page of content. Unlike reusable blocks, once a pattern is inserted it doesn't remain in sync with the original content as it is meant to be tweaked and customized by the user. Underneath the surface, patterns are just regular blocks composed together. Themes can register patterns to offer users quick starting points with a design language familiar to that theme's aesthetics.
```

Comment:
It might be helpful to dig deeper in _"it is meant to be tweaked and customized by the user."_ , as in it provides a structure jumpstart, and blocks inserted by a pattern can not also be edited but removed, or new blocks added to the composition.

Reply:
Thanks for the addition and keeping the reusable block information that remains relevant; I had originally added that list (to be removed) since patterns were not quite an established practice but they are now; so that can go :+1: